server/views/applications/pages/further-considerations/vulnerability.njk

```diff
@@ -0,0 +1,15 @@
+{% extends "../layout.njk" %}
+
+{% block questions %}
+  <h1 class="govuk-heading-l">
```

Comment:
I think semantically this would make more sense as a `<h1>` and a `<h2>`. 
To me they're two levels of heading but happy to bow to your wisdom if you think otherwise 🙇‍♂️

Reply:
I kinda agree with you, but this is an [established pattern](https://design-system.service.gov.uk/styles/typography/#headings-with-captions), so I'd rather not deviate from it. We can always tweak later.

Reply:
Well, every day is a school day!